saleor/graphql/account/mutations/account/account_register.py

```diff
@@ -188,3 +219,31 @@ def save(cls, info: ResolveInfo, user, cleaned_input):
 
             cls.call_event(manager.customer_created, user)
         account_events.customer_account_created_event(user=user)
+
+    @classmethod
+    def _clean_email_errors(cls, errors):
+        existing_user = False
+        filtered_errors = []
+
+        for error in errors:
+            if error.code == "unique":
+                existing_user = True
+                continue
+            filtered_errors.append(error)
+
+        return existing_user, filtered_errors
+
+    @classmethod
+    def _clean_errors(cls, error):
```

Comment:
Here as well.
I think it will be hard to recall after some time why it was added